redash/query_runner/mssql.py

```diff
@@ -0,0 +1,141 @@
+import json
+import logging
+import pymssql
```

Comment:
You should wrap the import of `pymssql` with a try/except block to avoid breaking existing deployments that don't have this package:

```python
try:
    import pymssql
    enabled = True
except ImportError:
    enabled = False
```

And then use the `enabled` value for the `enabled` method of the query runner:

```python
    @classmethod
    def enabled(cls):
        return enabled
```